process_manager/process_manager_interface.py

```diff
@@ -0,0 +1,104 @@
+"""Module providing functions to interact with the drunc process manager."""
+
+import asyncio
+from collections.abc import Iterable
+from enum import Enum
+
+from django.conf import settings
+from drunc.process_manager.process_manager_driver import ProcessManagerDriver
+from drunc.utils.shell_utils import DecodedResponse, create_dummy_token_from_uname
+from druncschema.process_manager_pb2 import (
+    LogRequest,
+    ProcessInstanceList,
+    ProcessQuery,
+    ProcessUUID,
+)
+
+
+def get_process_manager_driver() -> ProcessManagerDriver:
+    """Get a ProcessManagerDriver instance."""
+    token = create_dummy_token_from_uname()
+    return ProcessManagerDriver(
+        settings.PROCESS_MANAGER_URL, token=token, aio_channel=True
+    )
+
+
+async def _get_session_info() -> ProcessInstanceList:
+    pmd = get_process_manager_driver()
+    query = ProcessQuery(names=[".*"])
+    return await pmd.ps(query)
+
+
+def get_session_info() -> ProcessInstanceList:
+    """Get info about all sessions from process manager."""
+    return asyncio.run(_get_session_info())
```

Comment:
Is this what you meant by standardising how async methods are wrapped? Since this is a pattern that's re-used a number of times, it feels like maybe it's better suited to an OOP approach. But I think leaving it like this for now is fine until we make a decision on #140